image_provisioner/playbooks/bare_metal.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Prepare jump node
+  hosts: jump
+  user: root 
+  roles:
+    - ops_mirror_bootstrap
+    - qe_registry_bootstrap
+    - one-off
+    - create-local-repo
+    - docker-save
+    - dns-server-deploy
+
+- name: Allow root access
+  hosts: nodes
+  user: heat-admin
+  become: yes
+  roles:
+    - ssh-authorized-keys
```

Comment:
Typically we build in the root password, ssh keys, etc into the cloud config. So maybe we wanted to do it a different way, in the deployment of the base machines?